.github/workflows/build.yaml

```diff
@@ -0,0 +1,115 @@
+name: Build
```

Comment:
I dont know if it would be possible to make sure that two builds are not happening at the same time?